src/Screen.hh

```diff
@@ -526,7 +529,7 @@ private:
     std::unique_ptr<TooltipWindow> m_tooltip_window;
     FbTk::FbWindow m_dummy_window;
 
-    ScreenResource resource;
+    ScreenResource* m_resource;
```

Comment:
I'm not sure what the point is to make this a pointer? It only leads to more derefs, even in the only case where it's passed as a function parameter.

Reply:
I am guessing it was reflex that such a heavyweight thing shouldn't be "solid" and hence possibly copied around. I will leave it be in the re-do.